### ISSUE TITLE
Display warnings in SQ/SC UI when only test projects are analyzed

### ISSUE THREAD

**Issue by pavel-mikula-sonarsource** [SonarSource/sonar-dotnet#4063]
If there's at least one test file and zero main files, display warning in SC/SQ UI.

This warning will be coupled with #4052

**Comment by andrei-epure-sonarsource** [comment]
Verify WARN messages that start with `This sensor will be skipped`.
```
moq: This sensor will be skipped, because the current solution contains only TEST files and no MAIN files. Your SonarQube/SonarCloud project will not have results for vbnet files. You can read more about the detection of test projects here: https://github.com/SonarSource/sonar-scanner-msbuild/wiki/Analysis-of-product-projects-vs.-test-projects

nhibernate: This sensor will be skipped, because the current solution contains only TEST files and no MAIN files. Your SonarQube/SonarCloud project will not have results for vbnet files. You can read more about the detection of test projects here: https://github.com/SonarSource/sonar-scanner-msbuild/wiki/Analysis-of-product-projects-vs.-test-projects
```

Opened https://github.com/SonarSource/sonar-dotnet/issues/4095 as follow-up.
